CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## Next release
 
+## 0.32.0 / 2019-08-30
+
+* [FEATURE] Adding label selector for AlertManager objects discovery filtering (#2662)
```

Comment:
Could you order these as described in the release document? :slightly_smiling_face:  https://github.com/coreos/prometheus-operator/blob/master/RELEASE.md#prepare-your-release